src/schema/rules/entities.yaml

```diff
@@ -7,6 +7,8 @@
 - task
 - tracksys
 - acquisition
+- nucleus
+- volume
```

Comment:
What order should these be in? I think `nucleus` makes sense to group with `ceagent` and `tracer`, but `volume` doesn't seem to group here, intuitively. Possibly with `reconstruction` or `chunk`?

Reply:
Hi @effigies, in my opinion, `volume ` should be grouped with `sample`. `nucleus` should be grouped with `acq`.

Reply:
Placing `nucleus` between `acquisition` and `ceagent`/`tracer` sounds good to me. It seems like `volume` could fit well next to either `sample` or `chunk` (honestly, it seems like `sample` and `chunk` should be next to each other). I agree that `sample`'s a closer fit than `chunk` though.

Reply:
Sample is considered on par with subject and session (the microscopy BEP would have made sample first and subject optional, if it were feasible). I'm not sure that it holds the same place of priority here, but you may disagree.

Anyway, if we insert `volume` after `sample`, then the ordering for MRS would be:

```
sub-<label>[_ses-<label>][_voi-<label>][_task-<label>][_acq-<label>][_nuc-<label>][_rec-<label>][_run-<index>][_echo-<index>][_inv-<label>]_<suffix>.nii.gz
```

Lexically sorting file names, entities to the left will tend to indicate a bigger change than those to the right. Does this match your expectations when looking at a study?

Reply:
That's a really good point. Maybe it should go near chunk instead.

Reply:
Yeah, `voi` being there doesn't seem right to me either. I agree with @tsalo; I guess it belongs closer to `chunk`.

Reply:
Do you think before or after run? Do you think of it as doing multiple runs of multiple VOIs (`voi-<label>_run-<index>`) or multiple VOIs for each run (`run-<index>_voi-<label>`)?

Reply:
I would say the former: `voi-<label>_run-<index>`. In my opinion, the VOI (particularly for single-voxel spectroscopy [`svs`]), specifying `voi` is hugely important and should take precedence in file-naming.

Reply:
Okay. Then the current ordering is fine.